renku_notebooks/api/classes/k8s_client.py

```diff
@@ -115,6 +117,22 @@ def create_server(self, manifest: Dict[str, Any]) -> Dict[str, Any]:
         )
         return server
 
+    def patch_server(self, server_name: str, patch: Dict[str, Any]):
+        try:
+            status = self._custom_objects.patch_namespaced_custom_object(
+                group=self.amalthea_group,
+                version=self.amalthea_version,
+                namespace=self.namespace,
+                plural=self.amalthea_plural,
+                name=server_name,
+                body=patch,
+            )
+        except ApiException as e:
+            logging.exception(f"Cannot patch server {server_name} because of {e}")
+            raise PatchServerError()
+
+        return status
```

Comment:
It is fine to call this status... but I am pretty sure it is the manifest of the jupyterserver. And you can return it from this function and use it in other places. Esepecially since it would be nice if the patch method responded with the new server manifest rather than an empty body.